pkgs/tools/security/cfssl/default.nix

```diff
@@ -13,6 +13,10 @@ buildGoPackage rec {
     sha256 = "0j2gz2vl2pf7ir7sc7jrwmjnr67hk4qhxw09cjx132jbk337jc9x";
   };
 
+  # https://github.com/cloudflare/cfssl/pull/923 is merged
+  # remove this patch when it becomes part of a release.
+  patches = [ ./authkey-whitespace-trim.patch ];
```

Comment:
Can you use fetchpatch with https://github.com/cloudflare/cfssl/commit/7e13f60773c96644db9dd8d342d42fe3a4d26f36.patch ?

Reply:
oh. my review was too late.

Reply:
Not quite, you used the right commit hash. ;)